features/import.feature

```diff
@@ -87,3 +87,61 @@ Feature: Import content.
       """
       100
       """
+
+  Scenario: Export and import page and referencing menu item in separate files
+    Given a WP install
+    And I run `mkdir export`
+
+    When I run `wp menu create "My Menu"`
+    And I run `wp menu item add-post my-menu 2`
```

Comment:
If you use the `--porcelain` flag, then you can save the id for your new menu item to a local variable.

```
And I run `wp menu item add-post my-menu 2 --porcelain`
Then save STDOUT as {MENU_ITEM_ID}
```